data/json/items/gun/22.json

```diff
@@ -179,7 +180,11 @@
     "durability": 8,
     "valid_mod_locations": [ [ "accessories", 2 ], [ "barrel", 1 ], [ "bore", 1 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "muzzle", 1 ], [ "rail", 1 ], [ "sights", 1 ], [ "stock", 1 ], [ "underbarrel", 1 ] ],
     "magazine_well": 1,
-    "magazines": [ [ "22", [ "mosquitomag" ] ] ]
+    "magazines": [
```

Comment:
Very well then, removing the added magazine options and thus restoring this to how it was originally.